app.py

```diff
@@ -33,11 +35,13 @@ def post(self):
         response_input_lang = input_lang
 
         # Gets non-beta (fully supported) languages for automatic detection
-        fully_supported_lang_codes = ConverterRegistry.get_fully_supported_language_codes()
+        fully_supported_lang_codes = ConverterRegistry.get_fully_supported_language_codes(
+        )
```

Comment:
This seems to be wrong

Reply:
+1. This seems strange.

Reply:
Agreed, seems like a quirk of setting a fixed line cutoff. Not sure there's much to do about it, maybe could move the `.get_fully_...` to the next line.